package.json

```diff
@@ -61,6 +61,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "date-fns": "^2.29.3",
```

Comment:
Lets not add storybook dependencies to the project dependencies.

Reply:
+1 this adds to bundle size which we don't need